packages/system/src/use-style-config.ts

```diff
@@ -23,7 +23,7 @@ export function useStyleConfig(
   opts?: { isMultiPart?: boolean },
 ): SystemStyleObject
 
-export function useStyleConfig(themeKey: any, props: any, opts: any) {
+export function useStyleConfig(themeKey: any, props: any = {}, opts: any = {}) {
```

Comment:
Would the type `Record<string, any>` for `props` and `opts` help with this?

Reply:
I'm not sure it makes a difference. The `any` types are devoured by the overrides, and it seems like `Record<string, any>` and `any` both work fine with the defaults specified.

Reply:
Oh, I missed the overrides! 👍